src/internal/ui/prompt/tokenize_test.go

```
@@ -152,9 +152,8 @@ func Test_resolveShellSubstitution(t *testing.T) {
 		},
 		{
 			name:    "Multiple substitution",
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)